stories/Select.tsx

```diff
@@ -71,6 +72,66 @@ storiesOf('Organisms|Select Components', module)
             },
         }
     )
+    .addParameters(
+        StoreInjector.withStore({
```

Comment:
did you know that you can just use `React.useState` inside the story? I find this injector pattern very hard to read. I'm ok to leave it like this as you already wrote it, but maybe something to consider

Reply:
Cool, I'll reimplement with `useState `.

Reply:
Yes, with `useState` the code is cleaner. I think we can use `useState` for other stories. I returned back `addParameters` with `StoreInjector` and created a follow-up task for replacing it https://jira.textkernel.nl/browse/ONEUI-302